canmatrix/canmatrix.py

```diff
@@ -234,75 +106,48 @@ class Signal(object):
             _multiplex ('Multiplexor' or Number of Multiplex)
     """
 
-    def __init__(self, name, **kwargs):
-        self.mux_val = None
-        self.is_multiplexer = False
+    name = attr.ib(default = "")
+#    float_factory = attr.ib(default=defaultFloatFactory)
```

Comment:
@altendky  as you mentioned in #163,
should I just remove this complete here?

remove all float_factorys here in canmatrix.py

What to do with defaults than?
Or just use Decimal everywhere. Does Decimal have any thrawbacks?

Reply:
What about making decimal to the default for all format-modules also.
One could add an option to change the behaviour. 
But are there any Problems expected while using decimal instead of float?
Wouldn't it be the best to habe decimal everywhere in canmatrix?

What do you think?

Reply:
I assume there could be some unexpected behavior for people that want to use float but I haven't tried to dig into the details.  Generally though stuff like how decimal accentuates the oddities of float.

```python3
>>> decimal.Decimal(0.1)
Decimal('0.1000000000000000055511151231257827021181583404541015625')
```
But, I don't recall offhand where something like this would happen in canmatrix.  Mostly we would be creating decimals from text which works nicely.

Sorry I don't have a solid answer for this one.  :|  It seems like it would be fine to go pure decimal but I can't claim there won't be any valid complaints.